modules/installation-configuration-parameters.adoc

```diff
@@ -208,6 +207,11 @@ resources that it creates.
 For more information about AWS tags,
 see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html[Tagging Your Amazon EC2 Resources]
 in the AWS documentation.
+
+|`platform.aws.subnets`
+|If you provide the VPC instead of allowing the installation program to create the VPC for you, specify the subnets for the cluster to use. The subnets must be part of the same machineCIDR range that you specify and not used by any other process. For a standard cluster, specify a public and a private subnet for each availability zone. For a private cluster, specify a private subnet for each availability zone.
```

Comment:
> and not used by any other process.

I don't think that's true.

Reply:
So it's ok for a process outside of the cluster to try to use the subnet range?

Reply:
> So it's ok for a process outside of the cluster to try to use the subnet range?

Yup, you can share the subnets just like the VPC can be shared.  You may not want to share the CIDR, because tools would then be unable to route to both the subnet with that CIDR and whatever the other thing was.  But whatever, that's up to the user to decide if they would need simultaneous routing to both places or not.